docs/sites.yml

```diff
@@ -8005,3 +8005,15 @@
   built_by: Musthaq Ahamad
   built_by_url: "https://haxzie.com"
   featured: false
+- title: GBT
+  url: "https://yangmuzi.com/"
+  main_url: "https://yangmuzi.com/"
+  source_url: "https://github.com/yangnianbing/blog-by-gatsby"
+  description: >
+    It is a basic GatsBy site project
```

Comment:
```suggestion
    It is a basic Gatsby site project
```

spelling error.

Reply:
I have changed the spelling and made a new commit.
Please review once again.

Thanks